--- origsrc/wxLua/modules/wxbind/src/wxcore_bind.cpp	2008-02-02 11:25:57.000000000 -0600
+++ src/wxLua/modules/wxbind/src/wxcore_bind.cpp	2008-05-21 09:17:24.828125000 -0500
@@ -60,7 +60,7 @@
 wxCursor* wxLua_wxHOURGLASS_CURSOR = NULL;
 wxCursor* wxLua_wxCROSS_CURSOR = NULL;
 
-#if defined(__MINGW32__) || defined(__GNUWIN32__)
+#if defined(__MINGW32__) || defined(__GNUWIN32__) || defined(__CYGWIN__)
     // FIX: "internal compiler error: output_operand: invalid expression as operand"
     // We set their value again later since some compilers (gcc) won't set their value yet
     static wxPoint wxLua_wxDefaultPosition = wxDefaultPosition;
